app/components/Session/FormattedCaps.js

```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+import formatJSON from 'format-json';
+
+export default class NewSessionForm extends Component {
+
+  getCapsObject (caps) {
```

Comment:
this could be rewritten as i mentioned somewhere else as:

```js
let capObjs = caps.filter((cap) => !!cap.name).map((cap) => {[cap.name]: cap.value}));
return Object.assign({}, ...capObjs);
```